llvm/include/llvm/SandboxIR/Utils.h

```diff
@@ -0,0 +1,52 @@
+//===- SandboxIR.h ----------------------------------------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// Collector for SandboxIR related convenience functions that don't belong in
+// other classes.
+
+namespace llvm {
+
+namespace sandboxir {
```

Comment:
nit: `namespace llvm::sandboxir {`

Reply:
Done (incidentally, SandboxIR.h has it the other way.)